src/main/target/RACEBASE/readme.txt

```diff
@@ -0,0 +1,24 @@
+==RACABASE FC==
+
+Owner: Marcin Baliniak (marcin <at> baliniak.pl)
+
+Available at: shop.rotoracer.com
+
+Board information:
```

Comment:
Can you rename `readme.txt` to `README.md`

You can then use markdown styling.